docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1326,6 +1333,10 @@ An idempotent method to generate unique hashes per system is to use a salt that
     {{ 'secretpassword' | password_hash('sha512', 65534 | random(seed=inventory_hostname) | string) }}
     # => "$6$43927$lQxPKz2M2X.NWO.gK.t7phLwOKQMcSq72XxDZQ0XzYV6DlL1OD72h417aj16OnHTGxNzhftXJQBcjbunLEepM0"
 
+.. note::
+
+  While using ``seed`` parameter, the resulting value can be different depending on the version of Python used on the Ansible controller.
```

Comment:
```suggestion
  If you use the ``seed`` parameter, you will get a different result with Python 3 and Python 2. This may break procedures like password generation when you upgrade the version of Python used on your Ansible controller.
```